edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/ClientConnectionHandler.cs

```diff
@@ -21,18 +22,21 @@ namespace Microsoft.Azure.Devices.Edge.Hub.Amqp
     /// </summary>
     class ClientConnectionHandler : IConnectionHandler
     {
+        readonly TimeSpan closeTimeout = TimeSpan.FromSeconds(10);
```

Comment:
I know still draft, sorry for jumping the gun... but how this number was picked? maybe consider using "CloudOperationTimeout" setting here,

Reply:
I've checked the option to inject the timeout as a parameter, but it would go through several objects from the point of dependency registration and most of the time I had the feeling as "it does not belong there". Also, CloudOperationTimeout regards to cloud operations while this one is about talking to devices, so then the best would be to create its own (or a more general) timeout value.
Instead what I did, I checked in the amqp source code what it does and I saw that in the newer version there is a CloseAsync() with no timeout, which uses a hardcoded 60 sec timeout value. So I brought that value here and using that. When the time passes, the code throws a timeout exception